src/plugins/orangehrmAdminPlugin/Dao/LocalizationDao.php

```diff
@@ -83,4 +87,79 @@ public function saveI18NLanguage(I18NLanguage $i18NLanguage): I18NLanguage
         $this->persist($i18NLanguage);
         return $i18NLanguage;
     }
+
+    /**
+     * @param I18NTranslationSearchFilterParams $i18NTargetLangStringSearchFilterParams
+     * @return array e.g. [0 => ['id' => 1, 'source' => 'About', 'note' => null, 'target' => '关于']]
+     */
+    public function getNormalizedTranslations(
+        I18NTranslationSearchFilterParams $i18NTargetLangStringSearchFilterParams
+    ): array {
+        $q = $this->getTranslationsQueryBuilderWrapper($i18NTargetLangStringSearchFilterParams)->getQueryBuilder();
+        $q->select(
+            'langString.id',
```

Comment:
as we discussed we may have to add translation id,
```suggestion
            'translation.id',
```